src/star/star_solo/config.vsh.yaml

```diff
@@ -0,0 +1,115 @@
+name: star_solo
+namespace: star
+description: |
+  Aligns reads to a reference genome using STAR.
```

Comment:
This description is the same as align reads, I think. PErhaps be more descriptive?

Reply:
solved!